test/e2e/pvutil.go

```diff
@@ -446,6 +447,44 @@ func verifyGCEDiskAttached(diskName string, nodeName types.NodeName) bool {
 	return isAttached
 }
 
+// Sanity check for vSphere testing.  Verify the persistent disk attached to the node.
```

Comment:
this and next func move to vpshere specific e2e file, per general comment in conversation tab.